README.md

```diff
@@ -254,6 +254,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [GeoDataSource](https://www.geodatasource.com) — Location search service lookup for city name by using latitude and longitude coordinate. Free API queries up to 500 times per month.
   * [Glitterly](https://glitterly.app/) - Programatically generate dynamic images from base templates. Restful API and nocode integrations. Free tier comes with 50 images/month and 5 templates.
   * [GoodData](https://www.gooddata.com/) - Data as a Service - Create interactive and insightful dashboards. Free tier comes with 5 workspaces and 100 MB/workspace.
+  * [GridPanel](https://gridpanel.net/scraping-api) — Web Scraping API & mobile proxies. Helps you bypass anti-bot mechanisms and offers javascript rendering, mobile proxies, and geotargeting. Free tier of 1000 API calls.
```

Comment:
I merged this but now I am not sure - is this 1000 calls just a one off or a monthly allowance of 1000?

Reply:
Its 1000 calls. Not time-bound at all. You can use them at your leisure

Reply:
Then I am afraid its not going to qualify, just doesnt seem to compare.  Like I know they arent time bound but look at the other free tiers in this space - 2000 API calls a month etc.

So I'll revert this sorry

Reply:
Another in this list (ZenRows) has the exact same model:

> Free tier of 1000 API calls.

Reply:
Indeed, you can remove that one if you like, it should also not be added